test/extension/statusbartoggler.test.ts

```diff
@@ -1,13 +1,34 @@
-import {StatusBarToggler} from "../../src/extension/statusbartoggler";
+import * as assert from "assert";
+import { StatusBarToggler } from "../../src/extension/statusbartoggler";
 import { fakeConfig } from "../mocks/fakeConfig";
 
-suite("Status Bar Toggler Tests", function() {
-    test("Should toggle showStatusBarToggler command and message @unit", function() {
+suite("Status Bar Toggler Tests", () => {
+    test("Should toggle showStatusBarToggler command and message @unit", () => {
         const statusBarToggler = new StatusBarToggler(fakeConfig);
-        statusBarToggler.toggle();
+        assert.equal(statusBarToggler.statusText, "$(list-ordered) Watch");
+        statusBarToggler.toggle(true);
+        assert.equal(statusBarToggler.statusText, "$(list-ordered) Remove Watch");
     });
 
-    test("Should dispose when asked @unit", function() {
+    test("Should not toggle twice showStatusBarToggler command and message @unit", () => {
+        const statusBarToggler = new StatusBarToggler(fakeConfig);
+        assert.equal(statusBarToggler.statusText, "$(list-ordered) Watch");
+        statusBarToggler.toggle(true);
+        assert.equal(statusBarToggler.statusText, "$(list-ordered) Remove Watch");
+        statusBarToggler.toggle(true);
+        assert.equal(statusBarToggler.statusText, "$(list-ordered) Remove Watch");
+    });
+
+    test("Should toggle showStatusBarToggler command and message back to \"Wath\" @unit", () => {
```

Comment:
you probably meant `\"Watch"`

Reply:
```suggestion
    test("Should toggle showStatusBarToggler command and message back to \"Watch\" @unit", () => {
```

Thank you for the quick response. GitHub suggested me to use this tool that I can edit the code in the comment, If it doesn't work I'll update the MR as soon as I get home.

And thanks for the great extension you did! o/